pkg/apis/pipeline/pod/template.go

```diff
@@ -32,6 +32,13 @@ type Template struct {
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
+	// List of environment variables that can be provided to the containers belonging to the pod.
+	// +optional
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	// +listType=atomic
+	Envs []corev1.EnvVar `json:"envs,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,7,rep,name=envs"`
```

Comment:
NIT: This is called `env` in the k8s container API https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#container-v1-core. Since the naming of the rest of the fields is aligned with the container API, I think the extra "s" will be a source of confusion for users.